infra/terraform/test_fixtures/terraform-google-folders.tf

```diff
@@ -0,0 +1,68 @@
+locals {
+  // Roles required by CI service accounts to run the folders
+  // integration tests.
+  folders_required_roles = [
+    "roles/resourcemanager.folderCreator",
+  ]
+}
+
+resource "google_project" "folders" {
+  provider = "google.phoogle"
+  name = "ci-folders"
+  project_id = "ci-folders"
+  folder_id = "${google_folder.phoogle_cloud_foundation_cicd.name}"
+  billing_account = "${module.variables.phoogle_billing_account}"
+}
+
+resource "google_project_services" "folders" {
+  provider = "google.phoogle"
+
+  project = "${google_project.folders.id}"
+  services = [
+    "cloudresourcemanager.googleapis.com",
+  ]
+}
+
+resource "google_service_account" "folders" {
+  provider = "google.phoogle"
+  project      = "${google_project.folders.id}"
+  account_id   = "ci-folders"
+  display_name = "ci-folders"
+}
+
+resource "google_folder_folders_member" "folders" {
```

Comment:
This should be `google_folder_iam_member`.

Reply:
thanks will check this in other ci/cd pipelines I've PR-ed